restclient/RestClient.hx

```diff
@@ -9,60 +9,75 @@ import haxe.io.BytesOutput;
  */
 class RestClient
 {
+    #if flash
+    public static var useHttpOverride:Bool = true;
```

Comment:
I have an idea that might simplify this a bit. Instead of the extra #if check before calling setHeader, just make the setting in the case of flash constant:

```haxe
#if flash
public static inline var useHttpOverride:Bool = true;
#else
public static var useHttpOverride:Bool = false;
#end
```

Reply:
I thought about this some more. What this actually should be is something like "preferUseHttpOverride" and should have no effect in Flash case. We want client applications to be able to cross-compile against the library with minimal #if checks. If the application was setting useHttpOverride (my idea above), it would have to be in an #if block. On the other hand, just calling it useHttpOverride implies that the client has full control over whether the library uses the header (which isn't true in the Flash case).

Reply:
Would simply renaming "useHttpOverride" to "preferHttpMethodOverride" be good enough?

Reply:
I'll been thinking about the requestAsync() method:  it always uses Http.request(), so an override is always needed.  I'll push another commit for you to review.

Reply:
I've made the changes in eef217c